docs/modules/ROOT/pages/reference/billing-naming.adoc

```diff
@@ -0,0 +1,67 @@
+= Naming Scheme for Billing
+
+This page specifies various aspects of the AppCatalog billing.
+
+== Product Naming in the Database
+
+In order to support all future AppCatalog services and its subproducts, we need to define a naming scheme to differentiate between them.
+Each AppCatalog service will have multiple variations, let's take PostgreSQL as an example:
+
+* PostgreSQL standalone without SLA
+* PostgreSQL standalone with SLA
+* PostgreSQL replicated without SLA
+* PostgreSQL replicated with SLA
+* PostgreSQL CloudSQL
+* PostgreSQL RDS
+
+Every permutation will need an identifier, as it will be associated with a product in the https://kb.vshn.ch/appuio-cloud/references/architecture/metering-data-flow.html#_data_model[product dimension] of the billing database.
+Also, depending on the cluster where a given service is running, there could be different prices and discounts.
+
+To accommodate for all these variations, the default identifier of each product in the database should be as follows:
+
+ appcat_$servicename_$architecture_$sla:$clustername
+
+* `servicename`: The name of the AppCatalog service, in this example PostgreSQL
+* `architecture`: The architecture type of the service, for example `standalone` or `rds`.
+* `sla`: The name of the applied SLA.
+* `clustername`: The name of the cluster where this given instance is ordered.
+
+More specific product identifiers for special cases are added as needed.
+See the "examples" section of the https://kb.vshn.ch/appuio-cloud/references/architecture/metering-data-flow.html#_system_idea[system idea] for more information about how the matching works.
+
+== Query Names for Reporting
+
+The query names in the reporting tool should match the naming for the product.
+But it should not contain the `clustername` part.
+The cluster name is part of the `product` label that should be returned by the query.
+
+Examples for reporting query names:
+
+* appcat_postgresql_standalone_besteffort
+* appcat_redis_cluster_guaranteed
+* appcat_mariadb_rds
```

Comment:
@Following the naming convention, I think there should also be a name for the SLA, even if it's the one from the cloud provider. Any ideas what we might call it?
My ideas are
* besteffort
* passthrough
* 3rdparty
* cloud
* external
* standard
```suggestion
* appcat_mariadb_rds_besteffort
```

Reply:
There is no intention to call these SLAs differently than ours, we offer the same SLA also for cloud. Be aware that product management isn't well progressed, so it's subject to change anyways.

Reply:
In that case it would make sense to give it a name, in order to be better future-proof in case there will be different options.

Reply:
Use the same names as for all other services I'd say.